### PR TITLE
add simple examples and docs for row and column selection

### PR DESCRIPTION
**Pull request by r-brink**

### CODE REVIEW

File: user_guide/src/howcani/df/row_col_selection.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Row and column selection
+
+Selection of rows and columns is similar to other DataFrame libraries.
+
+## Column selection
+
+```python
+df[['a','b']]
```
Comment (carnarez) [inline user_guide/src/howcani/df/row_col_selection.md:8]: `"`

File: user_guide/src/examples/column_row_selection/__main__.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from ..paths import OUTPUT_BASE_DIR, create_if_not_exists
+from .dataset import df
+
+path = create_if_not_exists(f"{OUTPUT_BASE_DIR}/row_col_selection")
+
+with open(f"{path}/col_selection.txt", "w") as f:
+    f.write(f"{df[['a','b']]}\n")
+
+with open(f"{path}/row_selection.txt", "w") as f:
+    f.write(f"{df[0:2]}\n")
```
Comment (carnarez) [inline user_guide/src/examples/column_row_selection/__main__.py:10]: Not linted? `black` doesn't seem to have been ran on this.

File: user_guide/src/howcani/df/row_col_selection.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Row selection
+
+```python
+df[0:2]
+```
+
+```text
+{{#include ../../outputs/row_col_selection/row_selection.txt}}
+```
```
Comment (carnarez) [inline user_guide/src/howcani/df/row_col_selection.md:23]: Could add some more complicate slicing example: check the [`__getitem__`](https://github.com/pola-rs/polars/blob/master/py-polars/polars/frame.py#L473) code.

File: Makefile [hunk 4]
```diff
@@ -24,6 +24,7 @@ run: data
 	$(PYTHON) -m user_guide.src.examples.aggregate
 	$(PYTHON) -m user_guide.src.examples.conditionally_apply
 	$(PYTHON) -m user_guide.src.examples.df_manipulations
+	$(PYTHON) -m user_guide.src.examples.column_row_selection
```
Comment (carnarez) [inline Makefile:27]: ABCD 😏

File: user_guide/src/howcani/df/row_col_selection.md (hunk 3 above)
Comment (r-brink) [reply user_guide/src/howcani/df/row_col_selection.md:23]: Next iteration?

File: Makefile (hunk 4 above)
Comment (r-brink) [reply Makefile:27]: is that a thing?

File: user_guide/src/examples/column_row_selection/__main__.py (hunk 2 above)
Comment (carnarez) [reply user_guide/src/examples/column_row_selection/__main__.py:10]: My bad, did not read thoroughly enough.

File: user_guide/src/howcani/df/row_col_selection.md (hunk 3 above)
Comment (carnarez) [reply user_guide/src/howcani/df/row_col_selection.md:23]: @ritchie46 now or later?

Comment (ritchie46) [reply user_guide/src/howcani/df/row_col_selection.md:23]: Later is fine. Then this is alreay in the guide. :+1:

**Pull request merged**